hack/triage/config.yaml

```
@@ -0,0 +1,269 @@
+    name: Initial Triage
+    dedup: false
+    description: >
+      Check new issue and apply labels for kind, priority, area (optional).
+    rules:
+      - triage-needs-kind
+      - triage-needs-priority
+
+  - id: triage-refinement
+    name: Triage Refinement
+    dedup: false
+    description: >
+      Work on issue to get them to an actionable state.
+      When ready, apply triage/accepted and help of good-first issue label
+    rules:
+      - triage-stale-awaiting-more-evidence
+      - triage-stale-kind-support
+      - triage-updated
+      - triage-again
+      - triage-lifecycle-stale-or-rotten-priority-critical-urgent
+      - triage-lifecycle-stale-or-rotten-priority-important-soon
+      - triage-lifecycle-stale-or-rotten-priority-important-longterm
+      - triage-lifecycle-frozen
+      - triage-all
+
+  - id: actionable
+    name: Actionable
+    description: >
+      Issue to be worked.
+    dedup: false
+    rules:
+      - actionable-not-assigned-without-help-or-good-first-issue
+      - actionable-updated
+      - actionable-all
+
+  - id: fix-me
+    name: Fix-me!
+    description: >
+      Ensure we are using the labels in a consistent way.
+    dedup: false
+    rules:
+      - fixme-avoid-triage-needs-information
+      - fixme-avoid-kind-design
+      - fixme-kind-proposal-without-kind-feature
+      - fixme-kind-support-with-other-kinds
+      - fixme-triage-accepted-with-kind-support-or-priority-awaiting-more-evidence
+      - fixme-help-or-good-first-issue-before-triage-accepted
+      - fixme-lifecycle-frozen-with-priority-critical-urgent-priority-important-soon-priority-important-longterm
+
+rules:
+  # Phase: Initial triage
+
+  triage-needs-kind:
+    # WHY?: kinds need to be assigned by maintainers, let's do it!
+    name: "needs-kind"
+    resolution: "Add a kind/ label: api-change, bug, cleanup, deprecation, design, documentation, failing-test, feature, flake, proposal, regression, release-blocking, support"
+    type: issue
+    filters:
+      # TODO: Configure plugins so needs-kind label is applied + use it for this filter
+      - label: "!kind/.*"
+
+  triage-needs-priority:
+    # WHY?: kinds need to be assigned by maintainers, let's do it!
+    name: "needs-priority"
+    resolution: "Add a priority/ label: priority/critical-urgent > priority/important-soon > priority/important-longterm > priority/backlog > priority/awaiting-more-evidence"
+    type: issue
+    filters:
+      # TODO: Configure plugins so needs-priority label is applied + use it for this filter
+      - label: "!priority/.*"
+
+  # Phase: Triage finalization
+
+  triage-stale-awaiting-more-evidence:
+    # WHY?: issue authors are supposed to provide answers timely when an issue is labeled priority/awaiting-more-evidence, otherwise /close
+    name: "priority/awaiting-more-evidence not commented in the last 15 days"
+    resolution: "Consider if to close with /close, consider if to remove priority/awaiting-more-evidence label"
```

```suggestion
    resolution: "Consider if to close with /close or to remove priority/awaiting-more-evidence label"
```
(I assume it's mutual exclusive)